### PR TITLE
Enable 'idrac-wsman' interface for RAID Configuration

### PR DESCRIPTION
**Pull request by nasirkamal**
Enable 'idrac-wsman' interface in ironic.conf for inspect, management, power and raid. This is important to configure and manage Dell Power Edge servers.

This is related to PR #146, where I accidentally deleted the parent repository.

### CODE REVIEW

File: ironic.conf [hunk 1]
```diff
@@ -10,11 +10,11 @@ enabled_deploy_interfaces = direct,fake
 # NOTE(dtantsur): when changing this, make sure to update the driver
 # dependencies in Dockerfile.
 enabled_hardware_types = ipmi,idrac,irmc,fake-hardware,redfish,ibmc
-enabled_inspect_interfaces = inspector,idrac,irmc,fake,redfish
-enabled_management_interfaces = ipmitool,idrac,irmc,fake,redfish,idrac-redfish,ibmc
-enabled_power_interfaces = ipmitool,idrac,irmc,fake,redfish,idrac-redfish,ibmc
-enabled_raid_interfaces = no-raid,irmc,agent,fake
-enabled_vendor_interfaces = ipmitool,no-vendor,idrac,fake,ibmc
+enabled_inspect_interfaces = inspector,idrac,irmc,fake,redfish,idrac-wsman
```
Comment (dtantsur) [inline ironic.conf:13]: All occurrences of just `drac` should be removed in favour of `drac-wsman`.

File: ironic.conf [hunk 2]
```diff
@@ -10,11 +10,11 @@ enabled_deploy_interfaces = direct,fake
...
+enabled_management_interfaces = ipmitool,idrac,irmc,fake,redfish,idrac-redfish,ibmc,idrac-wsman
+enabled_power_interfaces = ipmitool,idrac,irmc,fake,redfish,idrac-redfish,ibmc,idrac-wsman
+enabled_raid_interfaces = no-raid,irmc,agent,fake,idrac-wsman
+enabled_vendor_interfaces = ipmitool,no-vendor,idrac,fake,ibmc,idrac-wsman
```
Comment (dtantsur) [inline ironic.conf:17]: We should have raised it earlier, but why do we need vendor interfaces? They're not exposed in BMO.

Comment (nasirkamal) [reply ironic.conf]: We got the PR #146 approved for RAID interface but I accidentally removed the parent repository.
Removed vendor_interface.

File: ironic.conf (hunk 1 above)
Comment (nasirkamal) [reply ironic.conf:13]: @jtwill98 @demonCoder95 can we do that?

Comment (nasirkamal) [reply ironic.conf]: Just added idrac-wsman interface for RAID. Which got already approved in PR #146 but I had to create new PR for technical issues.

File: ironic.conf (hunk 2 above)
Comment (nasirkamal) [reply ironic.conf]: Removed idrac-wsman interface for vendor interface.

**Comment by nasirkamal** [comment]
\assign

File: ironic.conf (hunk 1 above)
Comment (demonCoder95) [reply ironic.conf]: I agree with @dtantsur here. ``drac`` is the deprecated name, we should use ``idrac-wsman`` here.

**Comment by demonCoder95** [comment]
@maelk I think this is good to go now!

**Comment by demonCoder95** [comment]
Can we rerun the test on this? @maelk @dhellmann

**Comment by demonCoder95** [comment]
@russellb @dhellmann @maelk This is a patch similar to mine. all testing done, it's good to go as well!

**Comment by demonCoder95** [comment]
@dhellmann why hasn't this merged yet? Does it require an LGTM label in addition to the approve label?

**Comment by maelk** [comment]
The CI is currently broken after #162 went in. However, reverting it might break this PR ? Maybe the rework of enabling the bios interface could be done here too ?

**Pull request closed without merging**